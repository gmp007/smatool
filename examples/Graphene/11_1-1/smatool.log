2024-06-16 12:37:31 - INFO - SMATool calculation started at 12:37:31 on 2024-06-16
2024-06-16 12:37:31 - INFO - ==================================
2024-06-16 12:37:31 - INFO - Input file used in calculation:
2024-06-16 12:37:31 - INFO - code_type: VASP
2024-06-16 12:37:31 - INFO - use_saved_data: True
2024-06-16 12:37:31 - INFO - use_dnn_gan: False
2024-06-16 12:37:31 - INFO - strains: 0.01 0.8 0.01
2024-06-16 12:37:31 - INFO - yieldpoint_offset: 0.002
2024-06-16 12:37:31 - INFO - nlayers: 1
2024-06-16 12:37:31 - INFO - vdwgap: 3.5
2024-06-16 12:37:31 - INFO - dimensional: 2D
2024-06-16 12:37:31 - INFO - mode: DFT
2024-06-16 12:37:31 - INFO - scrystal: cubic
2024-06-16 12:37:31 - INFO - plusminus: False
2024-06-16 12:37:31 - INFO - use_schmid_factors: False
2024-06-16 12:37:31 - INFO - components: ['Tensile_x']
2024-06-16 12:37:31 - INFO - md_supercell: [1, 1, 1]
2024-06-16 12:37:31 - INFO - md_timestep: 500.0
2024-06-16 12:37:31 - INFO - indent_radius: 2.0
2024-06-16 12:37:31 - INFO - custom_options: {'potential_dir': '/home/chineduekuma/codes/vasp/PBE'}
2024-06-16 12:37:31 - INFO - job_submit_command: mpirun -n 2 vasp_std > log
2024-06-16 12:37:31 - INFO - structure_file: graphene.vasp
2024-06-16 12:37:31 - INFO - rotation: True
2024-06-16 12:37:31 - INFO - save_struct: True
2024-06-16 12:37:31 - INFO - slipon: True
2024-06-16 12:37:31 - INFO - slip_parameters: 100, 1-11
2024-06-16 12:37:31 - INFO - Input parameters from smatool.in ends
2024-06-16 12:37:31 - INFO - ==================================
2024-06-16 12:37:34 - ERROR - An error occurred: name 'K_Slack' is not defined
2024-06-16 12:37:34 - INFO - 
2024-06-16 12:37:34 - INFO - Results are saved in the various component files
2024-06-16 12:37:34 - INFO - SMATool general output is in smatool.out
2024-06-16 12:37:34 - INFO - Enthalpy information is in enthalpy.dat
2024-06-16 12:37:34 - INFO - Schmid factor saved in schmid_factor.dat
2024-06-16 12:37:34 - INFO - Work hardening rate is in xx_work_hardening_rate.dat
2024-06-16 12:37:34 - INFO - Structure info in VASP format at each strain saved in '.ss' file
2024-06-16 12:37:34 - INFO - Yield strength profile saved in the smatool_plots folder
2024-06-16 12:37:34 - INFO - 
2024-06-16 12:37:34 - INFO - Calculation done in 3.36
2024-06-16 12:37:34 - INFO - Well done! GOOD LUCK!
2024-06-16 12:37:34 - INFO - SMATool calculation Ended at 12:37:34 on 2024-06-16
