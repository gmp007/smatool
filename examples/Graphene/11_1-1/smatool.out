❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤
❤                            Results for TENSILE_X                     ❤
❤                                    using                             ❤
❤                             SMATool Version: 1.0                     ❤
❤                       VASP code is used as a calculator              ❤
❤                          to perform DFT simulations                  ❤
❤                Calculations started at 12:37:34 on 2024-06-16        ❤
❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤
+-====================================================================-+
|                    This is a 2D lattice                              |
+-====================================================================-+
                             Ultimate strain (ε) = 0.210                             
                         Ultimate strength (σ_s) = 36.672 N/m                        
                          Yield Strength (σ_y) = 17.684 N/m                          
                         Stiffness (K) = 4553.567  (E-A) N/m                         
                          Stiffness (K) = 300.797 (P-A) N/m                          
                                      Bp = 8.247                                     
                             Stiffness (E) = 303.360 N/m                             
                           Thickness of material = 3.430 Å                           
                            Resilience (U_r) = 0.155 J/m^3                           
                            Toughness (U_r) = 1.900 J/m^3                            
                      Energy storage capacity (E_c) = 5.613 MJ/L                     
                    Energy storage capacity (E_c) = 706.082 Wh/Kg                    
                              Ductility (%D) = 21.000 %                              
                          Ductility Index (%DI) = 250.000 %                          
                             Poisson's ratio (v) = 0.332                             
                           Shear modulus (G) = 113.881 N/m                           
                    Longitudinal sound velocity (V_l) = 23.400 m/s                   
                       Shear sound velocity (V_s) = 12.263 m/s                       
                        Mean sound velocity (V_m) = 15.361 m/s                       
                          Pwave velocity (V_p) = 24.448 m/s                          
                         Debye temperature (T_D) = 2562.512 K                        
                     First Lamé parameter (lambda) = 224.876 N/m                     
                       Second Lamé parameter (mu) = 113.881 N/m                      
                         Grüneisen parameter (gamma) = 1.989                         
                    Min thermal Conductivity (Slack) = 5.907 W/mK                    
                    Min thermal Conductivity (Clarke) = 4.565 W/mK                   
                   Min thermal  Conductivity (Cahill) = 12.701 W/mK                  
               Integrated min thermal Conductivity (Slack) = 1.493 W/mK              
+------------------------------------------------------------------------------+
|                                 * CITATIONS *                                |
| ---------------------------------------------------------------------------- |
|           If you have used SMATool in your research, PLEASE cite:            |
|                                                                              |
|              SMATool: Strength of materials analysis toolkit,                |
|                                 C.E. Ekuma,                                  |
|             Computer Physics Communications 300, 109189, (2024)              |
|                                                                              |
|                                                                              |
|        Material strength analyzer: Python-based computational toolkit        |
|          for tensile and indent strength of materials, C.E. Ekuma,           |
|                        www.github.com/gmp007/smatool                         |
+------------------------------------------------------------------------------+
