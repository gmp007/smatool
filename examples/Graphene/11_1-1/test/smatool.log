2024-06-16 12:37:03 - INFO - SMATool calculation started at 12:37:03 on 2024-06-16
2024-06-16 12:37:03 - INFO - ==================================
2024-06-16 12:37:03 - INFO - Input file used in calculation:
2024-06-16 12:37:03 - INFO - code_type: VASP
2024-06-16 12:37:03 - INFO - use_saved_data: True
2024-06-16 12:37:03 - INFO - use_dnn_gan: False
2024-06-16 12:37:03 - INFO - strains: 0.01 0.8 0.01
2024-06-16 12:37:03 - INFO - yieldpoint_offset: 0.002
2024-06-16 12:37:03 - INFO - nlayers: 1
2024-06-16 12:37:03 - INFO - vdwgap: 3.5
2024-06-16 12:37:03 - INFO - dimensional: 2D
2024-06-16 12:37:03 - INFO - mode: DFT
2024-06-16 12:37:03 - INFO - scrystal: cubic
2024-06-16 12:37:03 - INFO - plusminus: False
2024-06-16 12:37:03 - INFO - use_schmid_factors: False
2024-06-16 12:37:03 - INFO - components: ['indent_strength']
2024-06-16 12:37:03 - INFO - md_supercell: [1, 1, 1]
2024-06-16 12:37:03 - INFO - md_timestep: 500.0
2024-06-16 12:37:03 - INFO - indent_radius: 2.0
2024-06-16 12:37:03 - INFO - custom_options: {'potential_dir': '/home/chineduekuma/codes/vasp/PBE'}
2024-06-16 12:37:03 - INFO - job_submit_command: mpirun -n 2 vasp_std > log
2024-06-16 12:37:03 - INFO - structure_file: graphene.vasp
2024-06-16 12:37:03 - INFO - rotation: True
2024-06-16 12:37:03 - INFO - save_struct: True
2024-06-16 12:37:03 - INFO - slipon: True
2024-06-16 12:37:03 - INFO - slip_parameters: 110, 1-11
2024-06-16 12:37:03 - INFO - Input parameters from smatool.in ends
2024-06-16 12:37:03 - INFO - ==================================
2024-06-16 12:37:07 - ERROR - An error occurred: name 'K_Slack' is not defined
2024-06-16 12:37:07 - INFO - 
2024-06-16 12:37:07 - INFO - Results are saved in the various component files
2024-06-16 12:37:07 - INFO - SMATool general output is in smatool.out
2024-06-16 12:37:07 - INFO - Enthalpy information is in enthalpy.dat
2024-06-16 12:37:07 - INFO - Schmid factor saved in schmid_factor.dat
2024-06-16 12:37:07 - INFO - Work hardening rate is in xx_work_hardening_rate.dat
2024-06-16 12:37:07 - INFO - Structure info in VASP format at each strain saved in '.ss' file
2024-06-16 12:37:07 - INFO - Yield strength profile saved in the smatool_plots folder
2024-06-16 12:37:07 - INFO - 
2024-06-16 12:37:07 - INFO - Calculation done in 3.42
2024-06-16 12:37:07 - INFO - Well done! GOOD LUCK!
2024-06-16 12:37:07 - INFO - SMATool calculation Ended at 12:37:07 on 2024-06-16
