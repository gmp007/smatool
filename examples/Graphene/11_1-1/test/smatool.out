❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤
❤                         Results for INDENT_STRENGTH                  ❤
❤                                    using                             ❤
❤                             SMATool Version: 1.0                     ❤
❤                       VASP code is used as a calculator              ❤
❤                          to perform DFT simulations                  ❤
❤                Calculations started at 12:37:07 on 2024-06-16        ❤
❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤❤
+-====================================================================-+
|                    This is a 2D lattice                              |
+-====================================================================-+
                             Ultimate strain (ε) = 0.100                             
                         Ultimate strength (σ_s) = 20.666 N/m                        
                          Yield Strength (σ_y) = 16.599 N/m                          
                         Stiffness (K) = 75769.258  (E-A) N/m                        
                          Stiffness (K) = 567.797 (P-A) N/m                          
                                      Bp = 0.000                                     
                             Stiffness (E) = 283.221 N/m                             
                           Thickness of material = 3.430 Å                           
                            Resilience (U_r) = 0.145 J/m^3                           
                            Toughness (U_r) = 0.377 J/m^3                            
                      Energy storage capacity (E_c) = 1.506 MJ/L                     
                    Energy storage capacity (E_c) = 189.476 Wh/Kg                    
                              Ductility (%D) = 10.000 %                              
                           Ductility Index (%DI) = 66.667 %                          
                             Poisson's ratio (v) = 0.417                             
                            Shear modulus (G) = 99.946 N/m                           
                    Longitudinal sound velocity (V_l) = 29.694 m/s                   
                       Shear sound velocity (V_s) = 11.488 m/s                       
                        Mean sound velocity (V_m) = 15.152 m/s                       
                          Pwave velocity (V_p) = 30.425 m/s                          
                         Debye temperature (T_D) = 2527.705 K                        
                     First Lamé parameter (lambda) = 501.166 N/m                     
                       Second Lamé parameter (mu) = 99.946 N/m                       
                         Grüneisen parameter (gamma) = 2.836                         
                    Min thermal Conductivity (Slack) = 3.752 W/mK                    
                    Min thermal Conductivity (Clarke) = 4.411 W/mK                   
                   Min thermal  Conductivity (Cahill) = 15.131 W/mK                  
               Integrated min thermal Conductivity (Slack) = 1.526 W/mK              
+------------------------------------------------------------------------------+
|                                 * CITATIONS *                                |
| ---------------------------------------------------------------------------- |
|           If you have used SMATool in your research, PLEASE cite:            |
|                                                                              |
|              SMATool: Strength of materials analysis toolkit,                |
|                                 C.E. Ekuma,                                  |
|             Computer Physics Communications 300, 109189, (2024)              |
|                                                                              |
|                                                                              |
|        Material strength analyzer: Python-based computational toolkit        |
|          for tensile and indent strength of materials, C.E. Ekuma,           |
|                        www.github.com/gmp007/smatool                         |
+------------------------------------------------------------------------------+
